DotNetCasClient/Validation/CasSaml11Response.cs

```diff
@@ -98,7 +98,7 @@ private void ProcessValidAssertion()
         {
             protoLogger.Debug("Unmarshalling SAML response");
             XmlDocument document = new XmlDocument();
-            document.Load(new StringReader(_CasResponse));
+            document.LoadXml(_CasResponse);

```

Comment:
Can you remove this specific line change from the pull request and stick it inside a different pull request?

I say that because this PR has a title related to IDisposable and this one line isn't related to the other ones.  I'll then accept this PR.

As for this one XmlDocument load change which you'll put in another PR... I want to do some validation testing against different production CAS servers to validate nothing breaks.   I'm curious as to what the URL/name of the 3rd party CAS server was you used (for my future reference.)